run.py

```diff
@@ -102,70 +102,61 @@ def record_results(
 
 def model_state(dataset, country, state, starting_beds, interventions=None):
 
-    # Constants
-    start_time = time.time()
-    HOSPITALIZATION_RATE = 0.0727
-    HOSPITALIZED_CASES_REQUIRING_ICU_CARE = 0.1397
-    TOTAL_INFECTED_PERIOD = 12
+    # we should cut this, only used by the get_timeseries function, but probably not needed
     MODEL_INTERVAL = 4
-    POP = dataset.get_population_by_country_state(country, state)
+
     # Pack all of the assumptions and parameters into a dict that can be passed into the model
-    MODEL_PARAMETERS = {
-        # Pack the changeable model parameters
+    DATA_PARAMETERS = {
         "timeseries": dataset.get_timeseries_by_country_state(
             country, state, MODEL_INTERVAL
         ),
         "beds": dataset.get_beds_by_country_state(country, state),
-        "population": POP,
-        # 'projection_iterations': 25, # Number of iterations into the future to project
-        "projection_iterations": 80,  # Number of iterations into the future to project
-        "r0": r0,
-        "interventions": interventions,
-        "hospitalization_rate": HOSPITALIZATION_RATE,
-        "case_fatality_rate": 0.0109341104294479,
-        "hospitalized_cases_requiring_icu_care": HOSPITALIZED_CASES_REQUIRING_ICU_CARE,
-        # Assumes that anyone who needs ICU care and doesn't get it dies
-        "case_fatality_rate_hospitals_overwhelmed": HOSPITALIZATION_RATE
-        * HOSPITALIZED_CASES_REQUIRING_ICU_CARE,
-        "hospital_capacity_change_daily_rate": 1.01227,  # Equivalent to 1.05 per four days
-        "max_hospital_capacity_factor": 2.07,
-        "initial_hospital_bed_utilization": 0.6,
-        "model_interval": 4,  # In days
-        "total_infected_period": 12,  # In days
-        "rolling_intervals_for_current_infected": int(
-            round(TOTAL_INFECTED_PERIOD / MODEL_INTERVAL, 0)
-        ),
-        "estimated_new_cases_per_death": 32,
-        "estimated_new_cases_per_confirmed": 20,
-        # added for seird model
-        "incubation_period": 5,  # In days
-        "duration_mild_infections": 10,  # In days
-        "icu_time_death": 7,  # Time from ICU admission to death, In days
-        "hospital_time_recovery": 11,  # Duration of hospitalization, In days
-        # If True use the harvard parameters directly, if not calculate off the above
-        "use_harvard_params": True,
-        # If True use the parameters that make R0 2.4, if not calculate off the above
-        "fix_r0": False,
-        # If True use the harvard model inputs for inital conditions and N (recreate their graph)
-        "use_harvard_init": False,
+        "population": dataset.get_population_by_country_state(country, state),
+    }
+
+    MODEL_PARAMETERS = {
```

Comment:
Fwiw, I think this would be a great case for [dataclasses](https://docs.python.org/3/library/dataclasses.html)
can define it similar to a class:

```
from dataclasses import dataclass

@dataclass
class ModelParameters(object):
    model: str = "seir"
    use_harvard_params: bool = False
    fix_r0: bool = False

    @property
    def case_fatality_rate_hospitals_overwhelmed(self):
        return self.hospitalization_rate / self.hospitalized_cases_requiring_icu_care

# Generates defaults but overriding fix_r0
model_parameters = ModelParameters(fix_r0=True)
```

Definitely not advocating for the change here, but wanted to put the idea out there :).
If you were interested, it would definitely be better as it's own pr, but it does give a bit more structure and easier documentation!